packages/backend/src/graphql/schema.graphql

```diff
@@ -32,101 +29,110 @@ type Query {
-
-  "Create liquidity withdrawal from Open Payments payment pointer"
-  createPaymentPointerWithdrawal(
-    input: CreatePaymentPointerWithdrawalInput!
-  ): PaymentPointerWithdrawalMutationResponse
-
-  "Posts liquidity withdrawal"
+  "Post liquidity withdrawal. Withdrawals are two-phase commits and are approved via this mutation."
   postLiquidityWithdrawal(
     "The id of the liquidity withdrawal to post."
     withdrawalId: String!
   ): LiquidityMutationResponse
 
-  "Void liquidity withdrawal"
+  "Void liquidity withdrawal. Withdrawals are two-phase commits and are rejected via this mutation."
   voidLiquidityWithdrawal(
     "The id of the liquidity withdrawal to void."
     withdrawalId: String!
   ): LiquidityMutationResponse
 
-  "Create an Open Payments Quote"
-  createQuote(input: CreateQuoteInput!): QuoteResponse!
+  "Create a payment pointer"
+  createPaymentPointer(
+    input: CreatePaymentPointerInput!
+  ): CreatePaymentPointerMutationResponse!
 
-  "Create an Open Payments Outgoing Payment"
-  createOutgoingPayment(
-    input: CreateOutgoingPaymentInput!
-  ): OutgoingPaymentResponse!
+  "Add a public key to a payment pointer that is used to verify Open Payments requests."
+  createPaymentPointerKey(
+    input: CreatePaymentPointerKeyInput!
+  ): CreatePaymentPointerKeyMutationResponse
+
+  "Revoke a public key associated with a payment pointer. Open Payment requests using this key for request signatures will be denied going forward."
+  revokePaymentPointerKey(
+    "Internal id of key"
+    id: String!
+  ): RevokePaymentPointerKeyMutationResponse
 
-  "Create an internal Open Payments Incoming Payment"
+  "Create an internal Open Payments Incoming Payment. The receiver has a payment pointer on this Rafiki instance."
   createIncomingPayment(
     input: CreateIncomingPaymentInput!
   ): IncomingPaymentResponse!
 
-  "Create an external Open Payments Incoming Payment"
+  "Create an external Open Payments Incoming Payment. The receiver has a payment pointer on another Rafiki instance."
   createReceiver(input: CreateReceiverInput!): CreateReceiverResponse!
 
+  "Create an Open Payments Quote"
+  createQuote(input: CreateQuoteInput!): QuoteResponse!
+
+  "Create an Open Payments Outgoing Payment"
+  createOutgoingPayment(
+    input: CreateOutgoingPaymentInput!
+  ): OutgoingPaymentResponse!
+
   "Deposit webhook event liquidity"
   depositEventLiquidity(eventId: String!): LiquidityMutationResponse
 
   "Withdraw webhook event liquidity"
   withdrawEventLiquidity(eventId: String!): LiquidityMutationResponse
 
-  "Create payment pointer key"
-  createPaymentPointerKey(
-    input: CreatePaymentPointerKeyInput!
-  ): CreatePaymentPointerKeyMutationResponse
+  "Withdraw liquidity from Open Payments payment pointer prior to having received a webhook event. This can be used to e.g. withdraw funds from a partially completed incoming payment while still waiting for the remaining funds to be received."
```

Comment:
I _think_ this is what you would call when you receive a `payment_pointer.web_monetization` webhook event